website/common/locales/en/settings.json

```diff
@@ -16,6 +16,7 @@
     "startAdvCollapsed": "Advanced Options in tasks start collapsed",
     "startAdvCollapsedPop": "With this option set, Advanced Options will be hidden when you first open a task for editing.",
     "dontShowAgain": "Don't show this again",
+    "suppressLoginIncentiveModal": "Don't show popup when checking in",
```

Comment:
Yes, I prefer "Don't show "Check-In Counter" popup each morning" because it is more specific. Does this hide all of them, or only the ones between prizes?